redbot/core/events.py

```diff
@@ -97,10 +98,58 @@ async def on_ready():
                     "Outdated version! {} is available "
                     "but you're using {}".format(data["info"]["version"], red_version)
                 )
-                outdated_red_message = (
+                outdated_red_message = _(
                     "Your Red instance is out of date! {} is the current "
                     "version, however you are using {}!"
                 ).format(data["info"]["version"], red_version)
+                requires_python = data["info"]["requires_python"]
+                current_python = platform.python_version()
+                if expected_version(current_python, requires_python):
+                    extra_update = _(
+                        "\n\nTo update your bot, first shutdown your "
+                        "bot then open a window of {console} (Not as admin) and "
+                        "run the following:\n\n"
+                    ).format(
+                        console=_("Command Prompt")
+                        if platform.system() == "Windows"
+                        else _("Terminal")
+                    )
+                    extra_update += '```"{python}" -m pip install -U Red-DiscordBot```'.format(
+                        python=sys.executable
+                    )
+                    extra_update += _(
+                        "\n\nPlease be aware that the command above is a best guess;"
+                        "\nIt is based on your current OS, environment, and Python version so "
+                        "there is a small chance it could be incorrect."
+                        "\n**Should the above not work for you, please check our docs at <{docs}> "
+                        "and/or visit our support server at <https://discord.gg/red>.**"
+                    ).format(
+                        docs="https://docs.discord.red/en/stable/install_windows.html"
+                        if platform.system() == "Windows"
+                        else "https://docs.discord.red/en/stable/install_linux_mac.html",
+                    )
+
+                    outdated_red_message += extra_update
```

Comment:
This (and the other line exactly like this below) can both be replaced by one of this at the bottom, unindented once.